core/trino-main/src/main/java/io/trino/sql/planner/OptimizerConfig.java

```diff
@@ -291,6 +292,19 @@ public OptimizerConfig setFilterConjunctionIndependenceFactor(double filterConju
         return this;
     }
 
+    public boolean isUnestimatedFilterCostApproximationEnabled()
+    {
+        return unestimatedFilterCostApproximationEnabled;
+    }
+
+    @Config("optimizer.unestimated-filter-cost-approximation-enabled")
```

Comment:
convention is to use `.enabled` at the end

Reply:
There are 4 instances of `-enabled` and 3 instances of `.enabled` in this file.

Reply:
> There are 4 instances of -enabled and 3 instances of .enabled in this file.

I know, but `.enabled`. should be the suffix for new properties.